src/types.ts

```diff
@@ -95,6 +95,7 @@ export interface CompactOptions {
   removeEmptyObject?: boolean
   removeEmptyArray?: boolean
   compactArrays?: boolean
+  removeFn?: (val: any) => boolean
```

Comment:
You may wanna pass `node` as well. I've written code before that filters based on values and paths.

Reply:
Done.